extensions/amp-bind/0.1/test/test-bind-impl.js

```diff
@@ -56,9 +56,12 @@ describes.realWin('amp-bind', {
    * @return {!Element}
    */
   function createElementWithBinding(binding) {
+    const fakeDiv = env.win.document.createElement('div');
```

Comment:
The old version only ever created one element with a binding. I needed to change it to make one per call.